tmscti/callsupply.sqf

```diff
@@ -0,0 +1,73 @@
+hint "Your selected supply is on the way";
+
+_row = lbCurSel 1500;
+_supply = lbData [1500, _row];
+
+_CH47F = createVehicle ["RHS_CH_47F", getMarkerPos "sp_e", [], 0, "FLY"];
+
+createVehicleCrew (_CH47F);
+_cargo1 = _supply createVehicle getMarkerPos "sp_e";
+
+// For basecontainers and other "not slingloadable" objects:
+_cargo1 setMass [5000, 0];
+
+containerrope1 = ropeCreate [_CH47F, "slingload0", _cargo1, [0, 1.0, 1], 10];
+containerrope2 = ropeCreate [_CH47F, "slingload0", _cargo1, [0, -1.0, 1], 10];
+containerrope3 = ropeCreate [_CH47F, "slingload0", _cargo1, [3.0, 0, 1], 10];
+containerrope4 = ropeCreate [_CH47F, "slingload0", _cargo1, [-3.0, 0, 1], 10];
```

Comment:
These four cannot be global variables, else you may end up cutting the wrong ropes later if there is more than one delivery helicopter in the air at the time. Try setting these as variables on the helicopter itself, so you can access them from within the ```setWaypointStatements``` hook.